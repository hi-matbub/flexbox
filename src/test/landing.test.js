import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Landing from '../comps/landing.jsx';

it('renders smoke test without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Landing />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders snapshot as expected', () => {
  const tree = renderer.create(<Landing />).toJSON();
  expect(tree).toMatchSnapshot();
});
