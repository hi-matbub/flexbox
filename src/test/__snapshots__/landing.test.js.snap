// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot as expected 1`] = `
<header
  className="header"
>
  <div
    className="col-cont"
  >
    <div
      className="lfb-cont"
    >
      <span
        className="learnflexbox"
      >
        #
        <h1>
          learn
          <span>
            flexbox
          </span>
        </h1>
      </span>
      <p
        className="alt-text"
        style={
          Object {
            "marginBottom": "24px",
          }
        }
      >
        Subscribe for 30 days of Wire-framing Challenges.
      </p>
      <div
        id="mc_embed_signup"
      >
        <form
          action="https://matbub.us17.list-manage.com/subscribe/post?u=eb89624098c8ce40b7f3b26ef&id=909f807d9f"
          className="validate"
          id="mc-embedded-subscribe-form"
          method="post"
          name="mc-embedded-subscribe-form"
          noValidate={true}
          target="_blank"
        >
          <div
            className="form-cont"
          >
            <input
              aria-label="Repository description"
              className="email"
              defaultValue=""
              id="mce-EMAIL"
              name="EMAIL"
              placeholder="e.g. hi@matbub.co"
              style={
                Object {
                  "border": 0,
                }
              }
              type="email"
            />
            <div
              className="clear"
              id="mce-responses"
            >
              <div
                className="response"
                id="mce-error-response"
                style={
                  Object {
                    "display": "none",
                  }
                }
              />
              <div
                className="response"
                id="mce-success-response"
                style={
                  Object {
                    "display": "none",
                  }
                }
              />
            </div>
            <button
              className="button"
              id="mc-embedded-subscribe"
              name="subscribe"
              type="submit"
            >
              Subscribe
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>
  <div
    className="col-char"
  >
    <img
      alt="character"
      className="char"
      src="char-001.svg"
    />
  </div>
</header>
`;
